js/src/UXCapture.js

```diff
@@ -2,44 +2,51 @@ import ExpectedMark from "./ExpectedMark";
 import View from "./View";
 
 const NOOP = () => {};
+const INTERACTION_START_MARK = "interactionStart";
+const NAVIGATION_START_MARK = "navigationStart";
 
 export default class UXCapture {
   constructor() {
     this.onMark = NOOP;
     this.onMeasure = NOOP;
+    this.startMark = NAVIGATION_START_MARK;
```

Comment:
Just a heads up that I think @sergeychernyshev might be adding the babel plugin that allows us to use class properties for this kind of assignment. No need to do anything here, but we'll be able to clean up our constructors soon.

Reply:
I already added that plugin as of 2.0.2 (pushed this morning).

Might be a good idea to rewrite the code to use class props though - I'll file a ticket.